core/templates/domain/user/user-info.model.ts

```diff
@@ -98,6 +98,10 @@ export class UserInfo {
     return this._roles.includes('QUESTION_ADMIN');
   }
 
+  isQuestionCoordinator(): boolean {
+    return this._roles.includes('QUESTION_COORDINATOR');
```

Comment:
Optional: Can you create an enum for all roles? Like: 
```
enum UserRoles {
  QUESTION_COORDINATOR='QUESTION_COORDINATOR',
  QUESTION_ADMIN='QUESTION_ADMIN'
  ...
}
```
Then use it here:
`this._roles.includes(UserRoles.QUESTION_COORDINATOR)`

Reply:
Added the suggested enum. Sorry for not responding to the comment.